src/media/anime/episode.rs

```diff
@@ -294,3 +301,19 @@ impl Media for Episode {
         }
     }
 }
+
+pub fn fix_empty_episode_versions(episode: &mut Episode) {
```

Comment:
This function is for internal use only, it is better to be in `src/media/anime/util.rs` and marked as `pub(crate)`